### PR TITLE
fix: dbt requirements have moved to aspects-dbt

### PR DESCRIPTION
**Pull request by pomegranited**
and a malformed code block.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @pomegranited! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Review by bmtcril — APPROVED** [review]
Thanks!

File: docs/how-tos/dbt_extensions.rst [hunk 1]
```diff
@@ -80,6 +89,9 @@ Step 4. Add your custom transforms
 Here is where you will need an understanding of dbt, Clickhouse, Aspects' data schemas, and the specific transforms you
 want to create.
 
+If you need any python dependencies beyond what is provided by aspects-dbt, add these to a ``requirements.txt`` file at
+the top level of your repository.
+
```
Comment (pomegranited) [inline docs/how-tos/dbt_extensions.rst:94]: Note: [this change](https://github.com/openedx/tutor-contrib-aspects/pull/877/files#diff-ab9493272ed9d2c60f468bc12868616e669474e31d0bf99c168bbb855be046b5R26-R32) makes the `requirements.txt` file optional.

**Comment by openedx-webhooks** [comment]
@pomegranited 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-aspects&entry.867055334=https://github.com/openedx/openedx-aspects/pull/225&entry.1484655318=https://github.com/pomegranited&entry.752974735=2024-06-19+01:05&entry.1917517419=2024-06-20+13:39&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**